x-pack/plugins/ingest_pipelines/public/application/sections/pipelines_list/main.tsx

```diff
@@ -79,6 +83,37 @@ export const PipelinesList: React.FunctionComponent<RouteComponentProps> = ({
     redirectToPathOrRedirectPath(getListPath());
   };
 
+  const isManagedPipelines = (allPipelines: Pipeline[], isManaged: boolean) => {
+    return allPipelines.filter((pipeline) => {
+      return isManaged ? pipeline.isManaged : !pipeline.isManaged;
+    });
+  };
+
+  function filteredPipelines(allPipelines: Pipeline[]) {
+    let filteredItems: Pipeline[] = [];
+
+    if (filters.managed.checked === 'on')
```

Comment:
Do you think its worth to memoize the result of this function to avoid having to concatenate bunch of arrays on each render?